quic/s2n-quic-transport/src/transmission/application.rs

```diff
@@ -130,6 +133,9 @@ impl<'a, S: Stream, Config: endpoint::Config> Normal<'a, S, Config> {
 
             self.path_manager.on_transmit(context);
 
+            self.datagram_manager
+                .on_transmit(context, self.stream_manager);
+
             let _ = self.stream_manager.on_transmit(context);
```

Comment:
can you add a comment discussing datagram or stream sending data first. what is the default and how users might configure this behavior.